libs/wlgen/wlgen/rta.py

```diff
@@ -43,11 +43,20 @@ class Phase(_Phase):
     pass
 
 class RTA(Workload):
+    """
+    Class for creating RT-App workloads
+    """
 
     def __init__(self,
                  target,
                  name,
                  calibration=None):
+        """
+        :param target: Devlib target to run workload on.
+        :param name: Human-readable name for the workload
```

Comment:
Just to be picky... you almost always use a full stop at the end of each param description.
It's up to you if you want to be consistent... otherwise for me it's ok like that as well.
You already did a lot on the documentation side ;-)